profiles/export.cfg

```
@@ -0,0 +1,27 @@
+[buildout]
+extends = prod.cfg
+extensions += mr.developer
+auto-checkout +=
+    collective.exportimport
+
+[instance]
+eggs += 
+    collective.exportimport
+
+[sources]
+collective.exportimport = git https://github.com/collective/collective.exportimport.git
+
+[versions]
+attrs = 21.4.0
+backports.functools-lru-cache = 1.6.6
+beautifulsoup4 = 4.9.3
+functools32 = 3.2.3.post2
```

```suggestion
collective.exportimport = 1.12
functools32 = 3.2.3.post2
```

We can use the exportimport release instead of the source checkout. But a source checkout is also fine.

---

Oh, that's even better. I had something at the back of my head about some changes that were not released yet, but that might be outdated. Changed to using released version.